src/store/actions/social.js

```diff
@@ -1,9 +1,22 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
+  if (data) {
+    const sanitizedSocial = data.map(({ _id, url, fontawesome_icons }) => {
+      return {
+         _id, url, fontawesome_icons 
+      };
+    });
+    return sanitizedSocial;
+  } else {
+    return null;
```

Comment:
- no need for the else statment
- don't change the data type (return the data whatever it is)